dask_sql/physical/rel/logical/aggregate.py

```diff
@@ -1,3 +1,24 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2021, NVIDIA CORPORATION.
```

Comment:
We include this in the header by default, but you may request to remove it.

Reply:
I think in this case I would vote to remove it, if you can. Because with just the single copyright text from NVIDIA it looks like this is a NVIDIA-owned project (even though copyright != ownership). I am not sure what your policy is though - do you need to have some kind of mentioning of NVIDIA? I am happy to add a "authors" section or alike.